### PR TITLE
openapi: Add default values to the thing_mount_path parameters

### PR DESCRIPTION
**Pull request by averche**
For `generic_mount_path=true` passed to the openapi endpoint, the following changes will be applied in the `openapi.json` for each _mount_path parameter. These changes are needed for proper library code generation.

```diff
        {
          "name": "alicloud_mount_path",
          "description": "Path that the backend was mounted at",
          "in": "path",
          "schema": {
            "type": "string",
++          "default": "alicloud"
          },
--        "required": true
        }
        "post": {
++        "operationId": "postAuthAlicloudLogin",
--        "operationId": "postAuthAlicloud_mount_pathLogin",
```

### CODE REVIEW

**Review by AnPucel — APPROVED** [review]
Makes sens to me!

File: vault/logical_system.go [hunk 1]
```diff
@@ -4606,9 +4606,10 @@ func (b *SystemBackend) pathInternalOpenAPI(ctx context.Context, req *logical.Re
 						Description: "Path that the backend was mounted at",
 						In:          "path",
 						Schema: &framework.OASSchema{
-							Type: "string",
+							Type:    "string",
+							Default: strings.TrimRight(mount, "/"),
 						},
-						Required: true,
+						Required: false,
```
Comment (maxb) [inline vault/logical_system.go:4612]: @averche This change re-adds something we previously reverted... it violates the OpenAPI spec.

When "in" is "path", "required" MUST be true.

Comment (averche) [reply vault/logical_system.go:4612]: You are right, I'll put in a PR to fix this. Thanks!

Comment (averche) [reply vault/logical_system.go:4612]: #18955

**Pull request merged**